src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Server-side Sampling
+sidebar_order: 1
+description: "Learn about how you can control which transactions Sentry retains."
+---
+
+Server-side sampling lets you control what transactions Sentry retains by setting sampling rules and rates so you see more of the transactions you want to explore further, and less of the ones you don’t, without reconfiguring the SDK or redeploying your application.
+
+![The Server-side Sampling page with a list of rules](server-side-sampling.png)
+
+Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more events from high converting pages, critical API endpoints, or need to focus on latency issues from your latest release.
+
+Server-side sampling also makes your SDK and server-side sampling largely independent of each other. This helps if you're building native applications with many releases active at a given time, or you've already configured server-side sampling to target specific transactions, but the SDK blocks more of those transactions being sent to our servers.
```

Comment:
```suggestion
Server-side sampling also makes your SDK and your sampling configuration largely independent of each other. This helps if you're building native applications with many releases active at a given time. 
```

Reply:
I agree, I think this simplification makes sense and we also avoid repeating server-side too many times in one paragraph.